### ISSUE TITLE
-[MGLMapView accessibilityElementAtIndex:] performs poorly on Simulator

### ISSUE THREAD

**Issue by friedbunny** [mapbox/mapbox-gl-native#14324]
When running XCUITests, `-[MGLMapView accessibilityElementAtIndex:]` can occasionally consume 100% of the Simulator’s CPU. This can cause the tests to fail if the test harness was waiting for the app to idle before continuing.

![Screen Shot 2019-04-03 at 1 31 20 PM](https://user-images.githubusercontent.com/1198851/55511264-fc13c700-5614-11e9-83cf-33a8d02ecef9.png)

### Steps to reproduce

 Run XCUITests on Simulator, e.g., using `ios-sdk-examples`.

### Configuration

**Mapbox SDK versions:** `ios-v4.10.0-beta.1`, others
**Device/simulator models:** iPhone SE (iOS 10.3.1), others
**Xcode version:** 10.2

/cc @julianrex @1ec5

**Comment by 1ec5** [comment]
(Note that this performance issue is specific to the use of VoiceOver or UI testing, but the relevant code doesn’t run under normal circumstances.)

**Comment by 1ec5** [comment]
If neither our style support nor our detailed VoiceOver support is the focus of the test, I’d recommend changing to a minimal style, such as one-liner.json, that has no road layers to query and turn into accessibility elements.

**Comment by malaba** [comment]
What is `one-liner.json` ? And where to find it ? Thanks
